doc/source/parameters/unresolved_cfd-dem/void_fraction.rst

```diff
@@ -23,10 +24,11 @@ In this subsection, all parameters required for the calculation of the void frac
    set Function expression = 0.5 + 0.25*sin(pi*x)*sin(pi*y)
    end
      
-If the ``mode`` chosen is ``pcm``, then the void fraction is calculated using the Particle Centered Method. If it is set to ``qcm``, then the void fraction is calculated using the Quadrature Centered Method. In these methods, the remaining parameters are required:
+If the ``mode`` chosen is ``pcm``, then the void fraction is calculated using the Particle Centered Method. If it is set to ``qcm``, then the void fraction is calculated using the Quadrature Centered Method. If it is set to ``spm``, then the void fraction is calculated using the satellite point method (divided approach). In these methods, the remaining parameters are required:
```

Comment:
What do you think of naming it satellite point instead of spm inside the prm file? I think spm is not as usual.

Reply:
In the source code they are all named fully. But I think it is cleaner and easier for the user to insert the names as abbreviations in the prm file. All abbreviations are explained in the documentation